ui/app/routes/vault/cluster/access/leases/index.js

```diff
@@ -10,10 +10,7 @@ export default class LeasesIndexRoute extends Route {
   @service router;
 
   beforeModel(transition) {
-    if (
-      this.modelFor('vault.cluster.access.leases').get('canList') &&
-      transition.targetName === this.routeName
-    ) {
+    if (this.modelFor('vault.cluster.access.leases').canList && transition.targetName === this.routeName) {
```

Comment:
❓ These capability checks didn't seem to cause problems. I replaced this one while doing a global `.get(` search, but I can revert if we'd rather not.